docs/01_nodeos/03_plugins/chain_plugin/index.md

```diff
@@ -73,7 +73,8 @@ Config Options for eosio::chain_plugin:
                                         application config dir)
   --checkpoint arg                      Pairs of [BLOCK_NUM,BLOCK_ID] that 
                                         should be enforced as checkpoints.
-  --wasm-runtime wavm/wabt              Override default WASM runtime
+  --wasm-runtime eos-vm|eos-vm-jit      Override default WASM runtime (wabt)
```

Comment:
Yes that's correct, `wabt` is also an option, but I left it out since it's the default and it's also mentioned in the option description. I can add it back in if you want

Reply:
Not worth the trouble.